src/compiler/compile/render_dom/Block.ts

```diff
@@ -393,15 +393,15 @@ export default class Block {
 			${this.chunks.declarations}
 
 			${Array.from(this.variables.values()).map(({ id, init }) => {
-				return init
-					? b`let ${id} = ${init}`
-					: b`let ${id}`;
-			})}
+		return init
```

Comment:
this is weird

Reply:
Agreed. I can't even figure out how this is supposed to look.

This is the result of simply tightening all the warnings to errors, and then running auto-fix, not adding any more rules. As to what rule causes/controls this change, I don't know. Any ideas?